src/main/java/com/hyperwallet/clientsdk/util/HyperwalletMultipartUtils.java

```diff
@@ -0,0 +1,64 @@
+package com.hyperwallet.clientsdk.util;
+
+import com.hyperwallet.clientsdk.HyperwalletException;
+import com.nimbusds.jose.JOSEException;
+import org.apache.commons.lang3.StringUtils;
+import com.hyperwallet.clientsdk.model.HyperwalletVerificationDocument;
+import net.minidev.json.JSONObject;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.text.ParseException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class HyperwalletMultipartUtils {
+
+    public static List<Multipart> convert(HyperwalletVerificationDocument uploadData) throws IOException {
+
+        final String crlf = "\r\n";
+        List<Multipart> multipartList = new ArrayList<Multipart>();
+        Multipart multipart = new Multipart();
+        JSONObject document = new JSONObject();
+    try {
+        if (!StringUtils.isEmpty(uploadData.getType())) document.put("type", uploadData.getType());
```

Comment:
it can be extracted to a method

```
addDocumentValue(document, "type", uploadData.getType());
addDocumentValue(document, "country", uploadData.getCountry());
...

private void addDocumentValue(JSONObject document, String field, Object value) {
   if (!StringUtils.isEmpty(value)) {
      document.put(field, value);
   }
}
```